ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/BasicAction.java

```diff
@@ -3732,6 +3732,8 @@ else if (record.value() instanceof ExceptionDeferrer)
 
     //    private Mutex _lock = new Mutex(); // TODO
     private List<Throwable> deferredThrowables = new ArrayList<>();
+
+    protected boolean dynamic1PC = TxControl.dynamic1PC;
```

Comment:
I know TxControl caches the initial value of dynamic1PC but doing this is another layer of caching which makes it harder in the future to make dynamic1PC changeable at runtime.